pkg/providers/instancetype/instancetype.go

```diff
@@ -99,10 +99,7 @@ func (p *Provider) List(ctx context.Context, kc *corev1beta1.KubeletConfiguratio
 		return nil, err
 	}
 	// Get AvailabilityZones from EC2
-	availabilityZones, err := p.getAvailabilityZones(ctx)
-	if err != nil {
-		return nil, err
-	}
+	instanceTypeZones := p.getInstanceTypeZones(ctx, instanceTypeOfferings)
```

Comment:
```suggestion
	zones := p.getInstanceTypeZones(ctx, instanceTypeOfferings)
```

Might just be me, but I find this naming confusing since the function that we call with this data then re-maps this to call it zones and refers to `instanceTypeZones` for something else